src/test/java/com/hubspot/imap/BaseGreenMailServerTest.java

```diff
@@ -34,7 +34,7 @@ protected ImapClientConfiguration getImapConfig() {
         .hostAndPort(HostAndPort.fromParts("localhost", greenMail.getImap().getPort()))
         .useSsl(false)
         .connectTimeoutMillis(1000)
-        .tracingEnabled(false)
+        .tracingEnabled(true)
```

Comment:
Is this intentional to switch tracing default to be enabled?

Reply:
yes, I see no reason for tests not to have this